### ISSUE TITLE
[sort-order] Add sort order details in the ColumnSheet

### ISSUE THREAD

**Issue by frosencrantz** [saulpw/visidata#2649]
It would be helpful to see the sort order in the ColumnSheet. It would be nice if sort order details could be modified from the ColumnSheet.  Another option would be to have a separate sheet for showing/modifying the sort order of the columns in a sheet.

There are 2 bits that are useful for sort order, the order of the columns when performing the sort, and whether the column should be sorted, and the direction of the sort (ascending/descending).

**Comment by midichef** [comment]
I submitted a draft PR to handle the display aspect. I don't have a suggestion to handle changing the sort direction.

It'd be hard to set aside keystrokes on the `ColumnsSheet` that match all the column-sorting keys: `[`, `z[`, `]`, and `z]`. Maybe there could be a special key prefix for the `ColumnsSheet`, similar to `g` and `z` and `gz`, that would apply the keystrokes to the underlying column? So you'd go to a row that represents a `Column`, then press `prefix` + `[` to sort the column represented by that row.

**Comment by frosencrantz** [comment]
Thank you for working on this, @midichef!

I have not had a chance to try the PR.  I don't remember if I had any good ideas for how to edit the sort values from the column sheet.  I guess if there was a sheet only for sort details the ordering of the rows would be the order of the sorting, and there would be a column that was for the direction of the sort, but I don't have a good key stroke for that.  Maybe I was thinking it would be something like the options sheet with boolean values.

Another prefix is a reasonable suggestion for the ColumnSheet so we can keep the '[', ']' characters. I'm not sure what that prefix would be, and does it have a related mnemonic like 'g' and 'z', something "same/remote scope" (vs wider/narrower)?  

For some sheets like ColumnSheet, I guess I would be ok with some semantic changes to the keystrokes if there is a logic to it. 
So replace the defaults for some of the keystrokes.  All the commands would still exist, and are available via menu or long commands, but key binding is specific to the column sheet.  There are different things I want to do on a ColumnSheet vs a typical data sheet.  I don't typically sort on the ColumnSheet, at times the column names, but I haven't done a secondary sort.

**Comment by midichef** [comment]
To avoid changing keystrokes, we could add a column for "sort priority". It would indicate the 1st, 2nd, 3rd priority columns used for sorting. It would be editable with `edit-cell` and after each edit would be constrained to only hold positive sequential integers like `1`, `2`, `3`. And add another boolean-like column for "sort direction", which upon `edit-cell` (`e`) would flip between `asc` and `desc`.

But constraining the sort order to be sequential would make it a bit awkward to edit. If you edit a column to have priority `3` when no other priorities have been added, the constraint would have to change the value to `1`.

**Comment by frosencrantz** [comment]
Yes, I think that works. Also, if the sort priority already has values like 1, 2, 3, and the edit adds a `2`, it would cause the existing 2 and 3 to each increase to 3 4.  Not sure about a `ge` type edit, where multiple 2's were added.  In that case you would insert them before the existing 2 3, but have to renumber the new 2s  as 2, 3, 4 ...

**Comment by saulpw** [comment]
The values for sort priority don't have to start from 1 and be sequential; they only have to be numeric and unique (and maybe they don't even have to be numeric, just ordered).  This would be similar to the existing `keycol` field on the Column.

We could also avoid a second column for sort direction, using negative numbers to indicate a descending sort.  This might be too clever but I think it's more elegant than having two columns, at least without column groups.

**Comment by anjakefala** [comment]
This was added! Columns Sheet will now have a `sortorder`. `-` denotes reversed direction. Number denotes sorting priority.
